Dockerfile

```diff
@@ -10,11 +10,11 @@ RUN if [ "${RELEASE_MODE}" = true ]; then \
     else gradle build --exclude-task test -Dorg.gradle.project.version=${APP_VERSION}; fi
 
 # For ARM build use flag: `--platform linux/arm64`
-FROM --platform=$BUILDPLATFORM amazoncorretto:11.0.20
+FROM --platform=$BUILDPLATFORM amazoncorretto:21.0.1
 LABEL version=${APP_VERSION} description="EPAM ReportPortal. Auth Service" maintainer="Andrei Varabyeu <andrei_varabyeu@epam.com>, Hleb Kanonik <hleb_kanonik@epam.com>"
 ARG APP_VERSION=${APP_VERSION}
 ENV APP_DIR=/usr/app
-ENV JAVA_OPTS="-Xmx1g -XX:+UseG1GC -XX:InitiatingHeapOccupancyPercent=70 -Djava.security.egd=file:/dev/./urandom"
+ENV JAVA_OPTS="-Xmx1g -XX:+UseG1GC -XX:InitiatingHeapOccupancyPercent=70 -Djava.security.egd=file:/dev/./urandom  --add-opens=java.base/java.lang=ALL-UNNAMED"
```

Comment:
temporary workaround for compiler exception https://www.baeldung.com/java-illegal-reflective-access